packages/expo-dev-launcher/bundle/components/EmptyBranchesMessage.tsx

```diff
@@ -86,7 +86,7 @@ export function EmptyBranchesMessage({
               }>
               <View px="2.5" py="2">
                 <Button.Text weight="medium" color="tertiary">
-                  Create a branch
+                  Publish an update
```

Comment:
What do you think about changing "branch" -> "update" in the rest of the copy in this file? I think updates, rather than branches, are the more primitive and important object here as I don't think you can have a branch without an update.